### PR TITLE
8.7.1 release notes

### PR DESCRIPTION
**Pull request by ycombinator**
This PR generates the `8.7.1` release notes.  

The release notes were generated by running `elastic-agent-changelog-tool build --version 8.7.1`.  Then, all `changelog/fragments/*` files were deleted.

### CODE REVIEW

**Comment by pierrehilbert** [comment]
The content here is different from https://github.com/elastic/ingest-docs/pull/150/files
Should we adapt summary to match?

**Comment by ycombinator** [comment]
> The content here is different from https://github.com/elastic/ingest-docs/pull/150/files Should we adapt summary to match?

Yes, good catch @pierrehilbert.  I will update this PR here to match the contents of https://github.com/elastic/ingest-docs/pull/150/files (for Agent related items only, of course).  

Going forward, how do we prevent such a mismatch?  Is there any fixed ordering we should follow - e.g. the PR to the https://github.com/elastic/elastic-agent repo first, after which the PR to https://github.com/elastic/ingest-docs/ is created with matching content or vice versa?

File: changelog/8.7.1.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2218
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2185
+      timestamp: 1679422558
+      file:
+        name: 1679422558-feature-flags.yaml
+        checksum: f6a2c45b76df4b708f19d25ff569ab0394403eab
+    - kind: bug-fix
+      summary: Fix actions acks delay
```
Comment (cmacknz) [inline changelog/8.7.1.yaml:32]:
```suggestion
      summary: Fix action acknowledgements being delayed by up to 5 minutes.
```

I think this is clearer.

File: changelog/8.7.1.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      description: |
+        Fixes a bug that caused delayed "ack" response following Osquerybeat query
```
Comment (cmacknz) [inline changelog/8.7.1.yaml:34]:
```suggestion
        Fix for OSQuery live query results taking up to five minutes to show up in Kibana.
```

This matches the language in the known issue for this bug https://www.elastic.co/guide/en/fleet/current/release-notes-8.7.0.html

File: changelog/8.7.1.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2406
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2410
+      timestamp: 1680101776
+      file:
+        name: 1680101776-Fix-actions-acks-delay.yaml
+        checksum: 1ec1403420a07e550002cc162fdd1d92fae28dfa
+    - kind: bug-fix
+      summary: Respecting logging.level settings coming whether from Fleet UI or config file.
```
Comment (cmacknz) [inline changelog/8.7.1.yaml:46]:
```suggestion
      summary: Fix for the logging.level setting not being respected.
```

File: changelog/8.7.1.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        - https://github.com/elastic/elastic-agent/pull/2456
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2399
+        - https://github.com/elastic/elastic-agent/issues/2450
+      timestamp: 1680764383
+      file:
+        name: 1680764383-Respecting-logging.level-settings.yaml
+        checksum: b4c689aa41884020f23848adab7b586d886c3659
+    - kind: bug-fix
+      summary: An empty proxy from the policy don't override the proxy set by --proxy-url.
```
Comment (cmacknz) [inline changelog/8.7.1.yaml:61]:
```suggestion
      summary: An empty proxy from a Fleet agent policy no longer overrides the proxy set by --proxy-url.
```

File: changelog/8.7.1.yaml (hunk 2 above)
Comment (ycombinator) [reply changelog/8.7.1.yaml:34]: I think we should make the description text here the same as your suggestion in https://github.com/elastic/ingest-docs/pull/150#discussion_r1172004810?

Comment (ycombinator) [reply changelog/8.7.1.yaml]: > I think we should make the description text here the same as your suggestion in [elastic/ingest-docs#150 (comment)](https://github.com/elastic/ingest-docs/pull/150#discussion_r1172004810)?


Done in 8f76938cfa88714c08bce9e5357f9ba6ba09c606.

**Comment by ycombinator** [comment]
@cmacknz I applied your suggestions from the feedback and also updated some descriptions to match your suggestions from https://github.com/elastic/ingest-docs/pull/150.  Please re-review when you get a chance, thanks!

**Review by cmacknz — APPROVED** [review]

**Review by pierrehilbert — APPROVED** [review]
LGTM

File: changelog/8.7.1.yaml [hunk 5]
```diff
@@ -0,0 +1,111 @@
+version: 8.7.1
```
Comment (cmacknz) [inline changelog/8.7.1.yaml:1]: Belated comment, looks like we missed https://github.com/elastic/elastic-agent/commit/2d90292e4d1843e7dd2417ea42804f1788d77c7e here and in the official release notes

Comment (ycombinator) [reply changelog/8.7.1.yaml:1]: Indeed, we did.  Thanks for catching.  It looks like [the PR for that change](https://github.com/elastic/elastic-agent/pull/2448/files) didn't add a CHANGELOG fragment file, which is why it didn't get included here.  I'll make a follow up PR to directly add an entry to the `8.7.1.yaml` file.

Comment (ycombinator) [reply changelog/8.7.1.yaml:1]: > I'll make a follow up PR to directly add an entry to the `8.7.1.yaml` file.

https://github.com/elastic/elastic-agent/pull/2577

Comment (ycombinator) [reply changelog/8.7.1.yaml:1]: cc: @kilfoyle, for reviewing https://github.com/elastic/elastic-agent/pull/2577 and for incorporating the missed entry into the official release notes as well.

**Pull request merged**